packages/rightdown/src/commands/format.ts

```
@@ -1,170 +1,238 @@
+import { PrettierFormatter } from '../formatters/prettier.js';
+import { BiomeFormatter } from '../formatters/biome.js';
+import type { IFormatter } from '../formatters/base.js';
 
 interface FormatCommandArgs {
-  source?: string;
-  path?: string;
-  input?: boolean;
-  text?: string;
-  preset?: PresetName;
-  output?: boolean;
+  files?: string[];
```

ESLint

[error] Array type using 'string[]' is forbidden. Use 'Array<string>' instead.

(@typescript-eslint/array-type)